client/src/components/UpdateProfile/UpdateProfile.jsx

```diff
@@ -0,0 +1,209 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import DateFnsUtils from '@date-io/date-fns'
+
+import {
+  Avatar,
+  Button,
+  FormControl,
+  FormControlLabel,
+  FormLabel,
+  Grid,
+  IconButton,
+  Radio,
+  RadioGroup,
+  TextField,
+  Typography
+} from '@material-ui/core'
+import { KeyboardDatePicker, MuiPickersUtilsProvider, } from '@material-ui/pickers'
```

Comment:
extra coma

Reply:
Actually I have a lot of code styling issues. Maybe we changed code style config and I haven't updated it?

Reply:
I did not, and neither had errors with `npm run eslint-build`
am I using wrong command?

Reply:
I'm using `npx standart` and it shows plenty of problems